app/controllers/search-query-builder.js

```diff
@@ -38,15 +37,17 @@ export default Ember.Controller.extend({
              "LESS_THAN_EQUAL_TO",
              "GREATER_THAN",
              "GREATER_THAN_EQUAL_TO",
-//             "TERM",
+             "TERM"
 //             "TERMS".
     ],
-    aggregateTypes:["COUNT", "SUM", "AVERAGE", "CONTAINS", "STARTS_WITH", "ENDS_WITH"],
+    aggregateTypes:["COUNT", "SUM", "AVERAGE", "TERM", "CONTAINS", "STARTS_WITH", "ENDS_WITH"],
     aggregateSort:["VALUE_DESC", "VALUE_ASC", "COUNT_DESC", "COUNT_ASC"],
     aggregates:[],
     queryFilters:[],
     queryFilterOperators:["AND", "OR", "NOT"],
-    url: computed('searchType', function() {
+    profileQueryParameter: true,
```

Comment:
I know the default is true, but can you make the default here to false?  3rd parties are going to expect the pub api user/location object model and not the directory object models